tests/e2e/specs/wordpress/pluginActivation.js

```diff
@@ -38,7 +38,9 @@ describe('Plugin Activation', () => {
     await expect(page).toMatch("You're all set!");
     await expect(page).toMatch('Tell some stories.');
 
-    await percySnapshot(page, 'Plugin Activation');
+    await percySnapshot(page, 'Plugin Activation', {
+      percyCSS: `.plugin-version-author-uri, .amp-plugin-notice { display: none; }`,
```

Comment:
Same here, let's also hide anything dynamic on this page:

* Update bubble: `span.update-plugins { display: none !important; }` (yes, needs `!important`)
* WP version: `#footer-upgrade { display: none; }`
* Update notification: `.update-message`
* Navigation (changes when there are updates available): `.subsubsub`

Reply:
Should the above we global. WordPress version and plugin globals will be on all pages.

Reply:
Global [here](https://github.com/google/web-stories-wp/pull/4979/files#diff-a2a5376c3415156571e7a719c86331a9c7141a7011aeda34a667a4cbf7b43c6bR5-R8).

Reply:
Yeah makes sense 👍